tests/wpunit/includes/Standard_Source_Test.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace ISC\Tests\WPUnit;
+
+use ISC\Includes\Standard_Source;
+use phpDocumentor\Reflection\Types\Object_;
+
+/**
+ * Testing the Standard_Source class
+ */
+class Standard_Source_Test extends \Codeception\TestCase\WPTestCase {
+
+	/**
+	 * User ID
+	 *
+	 * @var int
+	 */
+	protected $user_id;
+
+	/**
+	 * Cleanup
+	 */
+	protected function tearDown(): void {
+		// Delete the user created in test_get_standard_source_text_for_attachment_post_author
+		if ( isset( $this->user_id ) ) {
+			wp_delete_user( $this->user_id );
+		}
+
+		// Reset modified options
+		delete_option( 'isc_options' );
+
+		// Call parent tearDown
+		parent::tearDown();
+	}
+
+	/**
+	 * Test the get_standard_source_text() method
+	 *
+	 * @return void
+	 */
+	public function test_get_standard_source_text() {
+		$this->assertEquals( '© http://isc.local', Standard_Source::get_standard_source_text() );
+	}
+
+	/**
+	 * Test the get_standard_source() method to see which option is selected by default (should be Custom Text)
+	 *
+	 * @return void
+	 */
+	public function test_get_standard_source() {
+		$this->assertEquals( 'custom_text', Standard_Source::get_standard_source() );
+	}
+
+	/**
+	 * Test the standard_source_is() method to see which option is selected by default (should be Custom Text)
+	 *
+	 * @return void
+	 */
+	public function test_standard_source_is() {
+		$this->assertTrue( Standard_Source::standard_source_is( 'custom_text' ) );
+	}
```

Comment:
**Suggestion:** Consider using data providers for your tests. This will make your tests more readable and maintainable, and allow you to easily add more test cases in the future. [maintainability]
```suggestion
 /**
  * @dataProvider standardSourceProvider
  */
 public function test_standard_source($expected, $method, $param = null) {
     $this->assertEquals($expected, Standard_Source::$method($param));
 }

 public function standardSourceProvider() {
     return [
         ['© http://isc.local', 'get_standard_source_text'],
         ['custom_text', 'get_standard_source'],
         [true, 'standard_source_is', 'custom_text'],
     ];
 }
```